pyfstat/mcmc_based_searches.py

```diff
@@ -1000,11 +1000,17 @@ def plot_chainconsumer(self, save_fig=True, label_offset=0.25, dpi=300, **kwargs
 
         samples_plt = self._scale_samples(samples_plt, self.theta_keys)
 
-        import chainconsumer
+        try:
+            import chainconsumer
+        except ImportError:
+            logging.warning(
+                "Could not import 'chainconsumer' package, please install it to use this method."
+            )
+            return
 
         c = chainconsumer.ChainConsumer()
         c.add_chain(samples_plt, parameters=labels)
-        c.configure(smooth=0, summary=False, sigma2d=True)
+        c.configure(smooth=0, summary=False, sigma2d=True, usetex=False)
```

Comment:
```suggestion
        # to avoid dependency on 'kpsewhich' TeX tool, we set usetex=False
        c.configure(smooth=0, summary=False, sigma2d=True, usetex=False)
```